hooks/command

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+export DOCKER_CLI_EXPERIMENTAL=enabled
+export AWS_PAGER=""
+
+build="${BUILDKITE_BUILD_NUMBER:-0}"
+repository="$BUILDKITE_PLUGIN_DYNAMODB_IMAGE_REPOSITORY"
+tables=()
+script_dir="$(dirname "${BASH_SOURCE[0]}")"
+local_dynamo_container_id=""
+local_dynamo_port=8000
```

Comment:
Feel free to merge first to unblock your testing, but can we explore using a dynamic port number? This is to avoid port conflicts when multiple Buildkite agents run on the same EC2 instance.

Reply:
Yep, I've actually got working locally already. Will push up a PR for it shortly after this goes in 😄